pkg/common/utils/mysql/mysql.go

```diff
@@ -127,6 +129,21 @@ func (db *DB) DecommissionBE(nodes []*Backend) error {
 	return err
 }
 
+func (db *DB) DropBE(nodes []*Backend) error {
+	if len(nodes) == 0 {
+		klog.Infoln("mysql DropBE BE node is empty")
+		return nil
+	}
+	nodesString := fmt.Sprintf(`"%s:%d"`, nodes[0].Host, nodes[0].HeartbeatPort)
+	for _, node := range nodes[1:] {
```

Comment:
I have learned source code. always deal like that
nodesStr := ""
for _, node := nodes {
     if nodesStr == "" {
          nodesStr = fmt.Sprintf("%s:%d", node.Host, node.HeartbeatPort)
          continue
     } 
     nodesStr = nodesStr + fmt.Sprintf("%s:%d", node.Host, node.HeartbeatPort)
}

the code will beautiful to read.